spec/support/multiline_literal_brace_layout_examples.rb

```diff
@@ -330,6 +330,44 @@ def construct(*args)
       end
     end
 
+    if [
+      RuboCop::Cop::Layout::MultilineArrayBraceLayout,
```

Comment:
This seems pretty odd. Probably we just need different contexts for both or something like this.

Reply:
You are right, I'm going to refactor these shared examples

Reply:
@bbatsov I just refactored the shared examples in order to avoid ifs. Please take a look and let me know if it's ok now.

Reply:
Hmm, I still see this `if` in the diff. :-)

Reply:
You are totally right, sorry. I'll fix it.

Reply:
Sorry man, now it's correct.